llvm/lib/Target/M68k/M68kInstrInfo.cpp

```diff
@@ -872,10 +889,10 @@ unsigned M68kInstrInfo::getGlobalBaseReg(MachineFunction *MF) const {
   //
   // NOTE
   // Normally M68k uses A5 register as global base pointer but this will
-  // create unnecessary spill if we use less then 4 registers in code; since A5
-  // is callee-save anyway we could try to allocate caller-save first and if
-  // lucky get one, otherwise it does not really matter which callee-save to
-  // use.
+  // create unnecessary spill if we use less then 4 registers in code; since
```

Comment:
This got moved when I ran `clang-format` on this file, can remove the change if necessary

Reply:
You probably should run with `git clang-format HEAD~1`: [How to Submit a Patch](https://llvm.org/docs/Contributing.html#id9).